### ISSUE TITLE
Native HTML elements using ariaLabel prop instead of :aria-label attribute

### ISSUE THREAD

**Issue by aasimsyed** [activist-org/activist#1537]
### Terms

- [x] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [x] I agree to follow activist's [Code of Conduct](.github/CODE_OF_CONDUCT.md)

### Behavior

I could be wrong, but wanted to open the issue just in case:

Native HTML <button> elements in sidebar components are using ariaLabel prop instead of :aria-label attribute, causing screen readers to receive literal i18n keys instead of translated text.

e.g. 

```html
<button class="focus-brand absolute bottom-1 right-1 z-10 flex rounded-md border border-black/80 bg-white/80 p-1 text-black/80 dark:border-white/80 dark:bg-black/80 dark:text-white/80" aria-label="i18n.components.sidebar_left_content_organization.edit_aria_label"></button>
```

Files affected:
- SidebarLeftContentOrganization.vue
- SidebarLeftContentGroupPage.vue  
- SidebarLeftContentEvent.vue

Fix: Change ariaLabel="i18n.key" to :aria-label="$t('i18n.key')" on native HTML elements.

The aria-label attribute in the HTML is displaying the literal key value instead of the translation. Making the above change seems to fix it.

**Comment by andrewtavis** [comment]
Thanks @aasimsyed! We'll get to the review soon :)

**Comment by andrewtavis** [comment]
Closed by #1538 🚀 Really great to have you back on the project, @aasimsyed! @nicki182 and I will do the review of the e2e PR this weekend. Please let us know if there's a day that could work for you to do a call 😊

**Comment by aasimsyed** [comment]
I'm free tomorrow all day Eastern Time from 12pm onwards.

Aasim Syed
________________________________

**Comment by andrewtavis** [comment]
Great! I'll message you on Matrix :)
